<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamEasyClass</class>
 <widget class="QWidget" name="StreamEasyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamEasy</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QComboBox" name="streamType">
     <item>
      <property name="text">
       <string>SerialPort</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TcpClient</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="setting_btn">
     <property name="text">
      <string>S</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Ntrip">
     <property name="text">
      <string>Ntrip</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="start_btn">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
